src/libraries/System.Collections.Immutable/src/System.Collections.Immutable.csproj

```diff
@@ -10,6 +10,7 @@ The System.Collections.Immutable library is built-in as part of the shared frame
 
   <ItemGroup>
     <Compile Include="Properties\InternalsVisibleTo.cs" />
+    
```

Comment:
Looks accidental?

Reply:
It was actually on purpose, just to separate general stuff from the frozen stuff.